client/src/components/Header/components/exposed-nav.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { radioLocation } from '../../../../../config/env.json';
+
+const ExposedNav = (): JSX.Element => {
+  const { t } = useTranslation();
+  return (
+    <nav className='exposed-nav'>
+      <ul className='exposed-nav-list'>
+        <li key='learn'>
+          <Link className='exposed-nav-elements' to='/learn'>
+            {t('buttons.curriculum')}
+          </Link>
+        </li>
```

Comment:
Do we need curriculum link in nav bar? I can think of its usage in the different parts freeCodeCamp, but I can't think if its usage here 🤔 
```suggestion
```

Reply:
Yes, but @ahmadabdolsaheb suggested that we need the curriculum link there. 
Check it out here : https://github.com/freeCodeCamp/freeCodeCamp/issues/41373#issuecomment-1293635587  
Now I do agree that we don't need the curriculum link in some places.

Reply:
I am not sure how I could do that. How would I know that the current page is the curriculum page.

Reply:
Using `params` https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams, but I don't think this is needed because freeCodeCamp.org is the curriculum, so you can cheat and just make it always active.

But I will wait until Ahmad is free to give feedback, because maybe there is a better way or code in codebase with that functionality already in it

Reply:
@ahmadabdolsaheb  what feedback do you have in mind.

Reply:
Because of the nature of the holiday session, replies will be delayed. I found his comment in the original issue, I hope that helps https://github.com/freeCodeCamp/freeCodeCamp/issues/41373#issuecomment-835953169

Reply:
Thanks for your patience. I will give this a review in the next 24 hours 🤞

Reply:
Great!

Reply:
When the curriculum link is clicked and is in focus it gets a background color of grey by default, I know that we want an underline when this link is active, but that is becoming a bit tricky for me. For now, I would like to resolve this conversation.